cmd/convox/templates/init/rails/bin/web

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
This file comes from the convox/rails image. You should add your server.pid line to there.